ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java

```diff
@@ -178,6 +179,8 @@ public void send(String message, int id) {
         new Buffer().writeUtf8(message));
     } catch (IOException e) {
       notifyWebSocketFailed(id, e.getMessage());
+    } catch (IllegalStateException e) {
+      notifyWebSocketFailed(id, e.getMessage());
```

Comment:
Please combine these two catch blocks like this, `(IOException | IllegalStateException e)`